code/game/machinery/doors/airlock.dm

```diff
@@ -988,11 +988,6 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/proc/toggle_lock(var/forced = 0)
 	return locked ? unlock() : lock()
 
-/obj/machinery/door/airlock/allowed(mob/M)
```

Comment:
What does this removal relate to?

Reply:
Because commands check access, this bit of code meant if you bolted a door via a command you could not then unbolt it, since the access check would always return false. As far as I can tell there's no other reason it should return false when bolted regardless, so I removed it.

Reply:
It's probably better to have network device datums use `check_access`, which is the more modern and saner version of `allowed`. They do the same thing except on some specific objs (like doors, here) where `allowed` carries legacy connotations I'm not familiar with.